packages/wrangler/src/pages/deploy.tsx

```diff
@@ -446,6 +446,8 @@ ${failureMessage}`,
 		pages_project: deploymentResponse.project_name,
 		deployment_id: deploymentResponse.id,
 		url: deploymentResponse.url,
+		alias: alias,
```

Comment:
NIT - you can just do

```suggestion
		alias,
```